README.md

```diff
@@ -73,64 +73,76 @@ fileBox3.toDataURL()
 
 ### 1. Load File in to Box
 
-#### 1.1 `fromFile(filePath: string): FileBox`
+#### 1.1 optional argumetns: 
```

Comment:
The `optional arguments` can be put under the `1. Load File into Box` above.

Do not add a new number here.